contrib/python-pint/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-pint"
+pkgver = "0.23"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+checkdepends = ["python-pytest", "python-pytest-benchmark"]
```

Comment:
isn't this missing appdirs/flexcache/flexparser

Reply:
what makes you suspect that?

Reply:
they're imported in the code and declared as dependencies in the project itself

Reply:
they don't seem to be declared in pyproject, and looking through the project they seem to be vendored in

Reply:
> they don't seem to be declared in pyproject  

requirements.txt

but yeah, only in the next version they're removed from _vendor